piqe_ocp_lib/api/tasks/operator_ops.py

```diff
@@ -77,14 +78,26 @@ def add_operator_to_cluster(self, operator_name, target_namespaces=[]):
 
         return True
 
+    def is_operator_installed(self, operator_name, operator_namespace):
+        """
+        Check if operator is installed and returned true or false
+        :param operator_name: name of the operator.
+        :param operator_namespace: namespace of the operator
+        rturn: object of values of csv
+        """
+        csv = self.csv
+        csv_resp = csv.get_cluster_service_version(operator_name, operator_namespace)
+        return csv_resp    
+      
+
     def delete_operator_from_cluster(self, operator_name: str, namespace: str) -> bool:
         """
         Uninstall an operator from a cluster
         :param operator_name: name of the operator
         :param namespace: name of the namespace the operator is installed
         :return: success or failure
         """
-        csv = ClusterServiceVersion(self.kube_config_file)
+        csv = self.csv
```

Comment:
Same overservation as above, no need to make a local copy of self.csv, it can be accessed directly here.